EC-84: Restructure eZ Commerce doc, part 2 (#2)
ezsystems/commerce-documentation: docs/guide/breadcrumbs/breadcrumbs_api/routesbreadcrumbsgenerator.md

# RoutesBreadcrumbsGenerator

`Silversolutions\Bundle\EshopBundle\Service\Breadcrumbs\RoutesBreadcrumbsGenerator`

> generates the breadcrumbs for the Symfony routes.

Reviewer: Two paragraphs up the name of the method was preceded by a word "Method..." For the sake of consistency I would choose either way, my suggestion is to follow this example, meaning to make the sentence begin with the name of the method only.

## When it is used

> If the `eZ\Publish\Core\MVC\Symfony\Routing\ChainRouter` matches the request, this generator is responsible.

Reviewer: The before the name of the router seems obsolete.

> If the `eZ\Publish\Core\MVC\Symfony\Routing\ChainRouter` matches the request, this generator is responsible.

Reviewer 2: ```suggestion
If the `eZ\Publish\Core\MVC\Symfony\Routing\ChainRouter` matches the request, Routes Breadcrumbs Generator is responsible.

> Since this is the case for nearly every request, this generator must be registered with one of the lowest priorities.

Reviewer 2: ```suggestion
Since this is the case for nearly every request, the generator must be registered with one of the lowest priorities.
```

## Rendering process

> This gnerator renders breadcrumbs if the route for the currently matched controller action

Reviewer: "genrator"->"generator"

> This gnerator renders breadcrumbs if the route for the currently matched controller action

Reviewer 2: ```suggestion
Routes Breadcrumbs Generator renders breadcrumbs if the route for the currently matched controller action
```
